src/plugins/presentation_util/common/labs.ts

```diff
@@ -34,6 +35,21 @@ export const projects: { [ID in ProjectID]: ProjectConfig & { id: ID } } = {
     }),
     solutions: ['dashboard'],
   },
+  [DOC_TABLE_LEGACY]: {
```

Comment:
dear @clintandrewhall, I've been migrating our `DOC_TABLE_LEGACY` settings to `labs.ts` of `presentation_util` to make it work. But I don't think that's a clean solution. This setting should live in the `discover` plugin IMO ... is there a way to assign or add lab projects inside the `discover` plugin?

Reply:
Not at the moment, no.  We didn't opt to create a "registration" for labs, so that the Project IDs and Solutions would be in TS and complete.  You're the first "external" use of Labs, so this is good feedback for a follow-up.

Reply:
thx, honored to be the first external one! the should be a baken in service!